quinn-proto/src/tests/mod.rs

```diff
@@ -603,9 +603,13 @@ fn zero_rtt_rejection() {
     pair.client.connections.clear();
     pair.server.connections.clear();
 
+    // Need to modify ALPN protocols in the existing client crypto config to enable resumption
```

Comment:
nit: IIRC the only thing that we really need to preserve here is the session cache, but this comment suggests otherwise.

Reply:
Extended the comment to more clearly explain the constraints.